### DISCUSSION TITLE
beets2mpd

### DISCUSSION THREAD

**Discussion by trapd00r** [beetbox/beets#4006] [Show and tell]
Noticed the 'Show & Tell' category is empty.

I love how beets queries work so I made [a bridge](https://github.com/trapd00r/beet2mpd) between beets and mpd.
No more 
`mpc search <tag> <query> | grep -v foo | grep -v bar`. 

:)

```
beet2mpd
beet2mpd comp:1
beet2mpd loved:1
beet2mpd genre:game
beet2mpd mbid_trackid::^$
beet2mpd label:'Masters of Hardcore' angerfist
beet2mpd year:2000..2004 ^genre:pop,rock,rap,hip-hop love
beet2mpd comments:foo # do note the plural form!
```

**Comment by sampsyo** [comment]
Just wanted to say that this is awesome!!

**Comment by doctorfree** [comment]
Very cool. I am using the Beets `play` plugin with `command: /usr/share/musicplayerplus/scripts/mpcplay.sh $args`. The `mpcplay.sh` script processes the arguments given the `beet play ...` command and executes `mpc` which adds the results of a Beets query to the MPD playlist. My approach does some of what `beets2mpd` does but I think beets2mpd may handle additional query results. Anyway, I just wanted to say this is very cool.

One question: I do not see any license for beets2mpd, just a copyright. I publish a project, [MusicPlayerPlus](https://github.com/doctorfree/MusicPlayerPlus#readme), which includes customized and extended Beets support. If I wanted to include `beets2mpd` in this open source project then it would have to be licensed in a compatible manner. Do you have any plans to release beets2mpd under an open source license?

Thanks for your efforts and sharing. I've also learned a bit by studying your Beets config.yaml.
